src/patternfly/components/Popover/popover-title.hbs

```diff
@@ -1,9 +1,22 @@
-<div class="pf-c-popover__title{{#if popover-title--IsIcon}} pf-m-icon{{/if}}{{#if popover-title--modifier}} {{popover-title--modifier}}{{/if}}"
+<div class="pf-c-popover__title{{#if popover-title--modifier}} {{popover-title--modifier}}{{/if}}"
+  {{#if popover-title--id}}
+    id="{{popover-title--id}}"
+  {{/if}}
   {{#if popover-title--attribute}}
     {{{popover-title--attribute}}}
   {{/if}}>
-  {{#if popover-title--IsIcon}}
-    {{> popover-title-icon}}
-  {{/if}}
+  {{#> wrapper popover-title-icon--type=(ternary popover-title-icon--type popover-title-icon--type
+        (ternary popover--IsDefaultAlert "bell"
+        (ternary popover--IsInfoAlert "info-circle"
+        (ternary popover--IsSuccessAlert "info-circle"
+        (ternary popover--IsWarningAlert "info-circle"
+        (ternary popover--IsDangerAlert "info-circle"
```

Comment:
Icons appearing again! We should update the success/warning/danger icons to their original, as they're all showing up with the info icon (albeit with the status color I think)

```suggestion
        (ternary popover--IsSuccessAlert "check-circle"
        (ternary popover--IsWarningAlert "exclamation-triangle"
        (ternary popover--IsDangerAlert "exclamation-circle"
```

Reply:
🤦‍♂️